src/picker/mod.rs

```diff
@@ -26,14 +81,56 @@ impl PickableItem {
         );
 
         PickableItem {
-            id: time_entry.id,
+            id: PickableItemId {
+                id: time_entry.id,
+                kind: PickableItemKind::TimeEntry,
+            },
             formatted: formatted_time_entry,
         }
     }
+
+    pub fn from_project(project: Project) -> PickableItem {
+        let formatted_project = format!(
+            "{}{}",
+            project.name,
+            match project.client.clone() {
+                Some(c) => format!(" - Client: {}", c.name),
+                None => "".to_string(),
+            }
+        );
+
+        PickableItem {
+            id: PickableItemId {
+                id: project.id,
+                kind: PickableItemKind::Project,
+            },
+            formatted: formatted_project,
+        }
+    }
+
+    pub fn from_task(task: Task) -> PickableItem {
+        let formatted_task = format!(
+            "{} (Task: {}){}",
+            task.project.name,
+            task.name,
+            match task.project.client.clone() {
+                Some(c) => format!(" - Client: {}", c.name),
+                None => "".to_string(),
+            }
+        );
+
+        PickableItem {
+            id: PickableItemId {
```

Comment:
I'm not too sure of the `id` nesting `id` struct, maybe the root id could be called a `guid`?
Minor though don't necessarily have to change.